locales/fr.json

```diff
@@ -0,0 +1,95 @@
+  "speech":"Aide vocale",
+  "condensed-sidebar":"Replier la colonne latérale",
+  "auto-hide-menubar":"Cacher automatiquement la barre de menu",
+  "new-group":"Nouveau groupe",
+  "title-asc":"Titre : Croissant",
+  "title-desc":"Titre : Décroissant",
+  "time-asc":"Heure : Croissant",
+  "time-desc":"Heure : Décroissant",
+  "add-group":"Ajouter un groupe",
+  "rename":"Renommer",
+  "move-to-root":"Déplacer vers la racine",
+  "move-to-group":"Déplacer vers un groupe",
+  "move-to-group-custom": "Déplacer vers {title}",
+  "move-to-trash":"Mettre à la corbeille",
+  "delete":"Supprimer",
+  "delete-permanently":"Suppression définitive",
+  "add-entry":"Ajouter un élément",
+  "search":"Chercher",
+  "select-or-create-an-entry":"Sélectionner ou créer un élément",
+  "save":"Enregistrer",
+  "empty-trash": "Vider la corbeille",
+  "empty-trash-question": "Êtes-vous sûr de vouloir vider cette corbeille ?",
+  "no-custom-fields-info-text":"Aucun champ personnalisé. Voulez-vous en ajouter un ?",
+  "custom-fields":"Champs personnalisés",
+  "title":"Titre",
+  "username":"Identifiant",
+  "secure-password":"Mot de passe sécurisé",
+  "untitled":"Sans titre",
+  "add-new-field":"Ajouter un nouveau champ",
+  "copy":"Copier",
+  "hide":"Cacher",
+  "reveal":"Afficher",
+  "reveal-password":"Affiche le mot de passe",
+  "installing":"En cours d’installation",
+  "update-available-message":"Buttercup {version} est disponible. Cliquez pour l’installer maintenant.",
+  "archive-saved-loading-info":"Votre archive est en cours de sauvegarde.<br>Fermeture automatique",
+  "unlock":"Déverrouiller",
+  "open":"Ouvrir",
+  "change-color":"Changer la couleur",
+  "import":"Importer",
+  "import-from-type":"Depuis l’archive {name} (.{extension})",
+  "import-to-type":"Vers {name}",
+  "archive-remove-with-name":"Retirer {name}",
+  "dropbox-connection-failed-info":"Échec de la connexion au serveur Dropbox. Veuillez réessayer ultérieurement",
+  "connect-to-dropbox":"Connexion à Dropbox",
+  "authenticate-with-dropbox":"Authentification avec Dropbox",
+  "dropbox-description-text":"Connectez Buttercup à votre compte Dropbox pour lire et enregistrer vos archives.<br />Nous n’enregistrons pas vos identifiants Dropbox.",
+  "webdav-connection-failed-info":"La connexion à {endpoint} a échoué. Veuillez vérifier vos identifiants et essayer à nouveau.",
+  "connect-to-wedav": "Connexion au serveur {title}",
+  "webdav-description-text":"Entrez l’adresse de connexion à {title} ainsi que votre identifiant et votre mot de passe puis choisissez une archive Buttercup. Nous <strong>enregistrons</strong> et <strong>encryptons</strong> vos identifiants.",
+  "name":"Nom",
+  "size":"Taille",
+  "date":"Date",
+  "open-url-in-browser": "Ouvrir l’adresse dans le navigateur",
+  "new-field": "Nouveau champ",
+  "label": "Label",
+  "archive": "Archive",
+  "file": "Fichier",
+  "archive-not-found-error": "La source de l’Archive est introuvable.",
```

Comment:
not need for having 'archive' capitalised

Reply:
Right